src/Microsoft.Identity.Client/TokenCache.cs

```diff
@@ -111,11 +117,11 @@ internal void SetServiceBundle(IServiceBundle serviceBundle)
         /// <param name="serviceBundle"></param>
         /// <param name="legacyCachePersistenceForTest"></param>
         internal TokenCache(IServiceBundle serviceBundle, ILegacyCachePersistence legacyCachePersistenceForTest)
+            : this(serviceBundle.PlatformProxy)
```

Comment:
why not pass service bundle instead of the platform proxy, you are setting service bundle here but might as well be done from this(...)?